### PR TITLE
Apply Conan Conventions

### PR DESCRIPTION
**Pull request by uilianries**

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -1,110 +1,85 @@
...
+            configure_args = ['--disable-examples']
+            if self.options.shared:
+                configure_args.extend(['--disable-static', '--enable-shared'])
+            else:
+                configure_args.extend(['--disable-shared', '--enable-static'])
+            self._autotools.configure(args=configure_args)
+        return self._autotools
 
     def build(self):
+        tools.patch(base_path=self._source_subfolder, patch_file="theora.patch")
```
Comment (danimtb) [inline conanfile.py]: If the patch is done fro every build, why not apply it in `source()`?

**Review by danimtb — APPROVED** [review]

File: .travis.yml [hunk 2]
```diff
@@ -41,13 +43,13 @@ matrix:
...
 install:
-  - chmod +x .travis/install.sh
-  - ./.travis/install.sh
+  - chmod +x .ci/install.sh
+  - ./.ci/install.sh
 
 script:
-  - chmod +x .travis/run.sh
-  - ./.travis/run.sh
+  - chmod +x .ci/run.sh
```
Comment (danimtb) [inline .travis.yml:54]: note that you change this path but scripts are still in `.travis` folder

File: conanfile.py (hunk 1 above)
Comment (uilianries) [reply conanfile.py]: I prefer to keep `source` as clean as possible. When I need to investigate the original project, I read the files from source, or when I need to apply a patch and check it consistence, I use the source folder for that.

File: .travis.yml (hunk 2 above)
Comment (uilianries) [reply .travis.yml:54]: Yeah, I made some shit here, it's all broken on CI :fire:

File: conanfile.py (hunk 1 above)
Comment (danimtb) [reply conanfile.py]: ok, np 👍

**Comment by uilianries** [comment]
Finally working!

**Pull request merged**